CONTRIBUTING.adoc

```diff
@@ -0,0 +1,27 @@
+[[contribution-guidelines]]
+= Contribution guidelines
+
+[[filing-issues]]
+== Filing issues
+
+File issues using the standard
+https://github.com/minishift/minishift-addons/issues[Github issue tracker] for the repository.
+Before you submit a new issue, we recommend that you search the list of issues to see if anyone already submitted a similar issue.
+
+[[contributing-patches]]
+== Contributing patches
+
+Thank you for your contributions! Please follow this process to submit a patch:
+
+.  Create an issue describing your proposed change to the repository.
+.  The repository owners will triage and respond to your issue promptly.
+.  Fork the repository and create a topic branch.
+.  Refer to the link:https://docs.openshift.org/latest/minishift/contributing/developing.html[developer documentation] for guidelines and tips on how to build and test Minishift.
```

Comment:
I guess you took this from the main Minishift repo. In this repo it us not relevant though.